packages/base-crypto/src/VirgilCrypto.ts

```diff
@@ -643,4 +673,12 @@ export class VirgilCrypto implements ICrypto {
     }
     return this.createHash(serializedPublicKey, this.foundationModules.Sha512).slice(0, 8);
   }
+
+  private throwUnlessDisposed() {
```

Comment:
`Unless - except if (used to introduce a case in which a statement being made is not true or valid).`

In this is case you throw error if instance is disposed.